src/components/bcros/filing/common/HeaderActions.vue

```diff
@@ -2,6 +2,7 @@
   <div class="flex flex-row items-center">
     <!-- the main button -->
     <UButton
+      v-if="!isBootstrapFiling || !isExpanded"
```

Comment:
Add this v-if logic: for bootstrap filing history, once the 'View' button is clicked and the content is expanded, the button becomes hidden.

Reply:
Clicking the button will toggle the value of `isExpanded`.  For regular filing, `!isBootstrapFiling` is always true so the button will always be displayed; it just shows different text (View vs Hide) based on the value of `isExpanded`. For bootstrap filing, `!isBootstrapFiling` is always false, so only the 'View' button is displayed, there is no 'Hide' button.